### DISCUSSION TITLE
Anyone know what's going on with Nils?

### DISCUSSION THREAD

**Discussion by useronkel** [nilsteampassnet/TeamPass#4269] [General]
Does anyone know what's about Nils?
Now gone for nearly 3 months, no Pull Requests edited, no code updates, no showing at all.

I'm just a little curious about his status

**Comment by DobroFenix** [comment]
What is the latest working version, which is installed without errors and all the functionality works?

**Comment by nilsteampassnet** [comment]
Hello @useronkel
Thank you for answering.
I had a kind of "too much" that required from me to stop all my extra activities, focusing more on myself.
Did some personal life cleaning and organized myself better with all those activities.
Back to teampass with a real wish to make it better but with an other engagement 😉
Cheers

**Reply by kcbieng** [reply]
Glad you're back! I was wondering if you were ok as well.
